internal/clients/ec2/ec2_client.go

```diff
@@ -34,8 +34,13 @@ func NewEC2Client(ctx context.Context) *Client {
 	cache := aws.NewCredentialsCache(credentials.NewStaticCredentialsProvider(
 		config.AWS.Key, config.AWS.Secret, config.AWS.Session))
 
+	region := params[0]
+	if region == "" {
+		region = config.AWS.Region
+	}
```

Comment:
The OpenAPI says it is required, but here we do have a default value. So what was the intention?

Reply:
Oh this is the client code, it is required in the service function, okay! I get it.